app/src/components/code/JSONToolEditor.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { json, jsonLanguage, jsonParseLinter } from "@codemirror/lang-json";
+import { linter } from "@codemirror/lint";
+import { EditorView, hoverTooltip } from "@codemirror/view";
+import { nord } from "@uiw/codemirror-theme-nord";
+import CodeMirror, { ReactCodeMirrorProps } from "@uiw/react-codemirror";
+import {
+  handleRefresh,
+  jsonCompletion,
+  jsonSchemaHover,
+  jsonSchemaLinter,
+  stateExtensions,
+} from "codemirror-json-schema";
+
+import { useTheme } from "@phoenix/contexts";
+import { toolJSONSchema } from "@phoenix/schemas/toolSchema";
+
+export type JSONToolEditorProps = Omit<
+  ReactCodeMirrorProps,
+  "theme" | "extensions" | "editable"
+>;
+
+export function JSONToolEditor(props: JSONToolEditorProps) {
+  const { theme } = useTheme();
+  const codeMirrorTheme = theme === "light" ? undefined : nord;
```

Comment:
I added a GitHub theme you can use for light mode syntax highlighting. See TemplateEditor for the exact import

Reply:
nice, ty would have copy pasta'd the regression here too